### PR TITLE
meta: Update README and other meta things.

### LINKED ISSUE #344: Take a pass over README and verify it is up-to-date.

### PR DESCRIPTION
**Pull request by HansKristian-Work**
Fix #344.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,11 +9,48 @@ The original project is available at [WineHQ](https://source.winehq.org/git/vkd3
...
+- Vulkan 1.1
+- `VK_EXT_descriptor_indexing` with at least 1000000 UpdateAfterBind descriptors for all types except UniformBuffer.
+  Essentially all features in `VkPhysicalDeviceDescriptorIndexingFeatures` must be supported.
+- `VK_KHR_timeline_semaphore`
+
+Some notable extensions that **should** be supported for optimal or correct behavior.
+These extensions will likely become mandatory later.
+
+- `VK_EXT_robustness2`
+- `VK_KHR_buffer_device_address`
```
Comment (doitsujin) [inline README.md:30]: Also, `VK_EXT_extended_dynamic_state`.

File: README.md [hunk 2]
```diff
@@ -9,11 +9,48 @@ The original project is available at [WineHQ](https://source.winehq.org/git/vkd3
...
+
+### AMD (RADV / ACO)
+
+For AMD, RADV is the recommended driver and the one that sees most testing on AMD GPUs.
+The recommendation here is to use a driver built from Git.
```
Comment (doitsujin) [inline README.md:35]: Anything known issues with 20.2?

File: README.md [hunk 3]
```diff
@@ -9,11 +9,48 @@ The original project is available at [WineHQ](https://source.winehq.org/git/vkd3
...
+> Reduce host memory consumption for descriptor memory when VkDescriptorSetVariableDescriptorCountAllocateInfo is used.
+
+> Fixed a bug in a barrier optimization that allowed some back-to-back copies to run unordered
+
+These fixes should find their way into stable drivers eventually, but if you're having issues, test the latest development drivers,
+as that is what we test against.
+
+### Intel
+
+We have not done any testing against Intel iGPU yet.
```
Comment (Joshua-Ashton) [inline README.md:52]: s/iGPU/iGPUs/

File: README.md [hunk 4]
```diff
@@ -24,78 +61,80 @@ git clone --recursive https://github.com/HansKristian-Work/vkd3d-proton
...
 ```
 
-## Using VKD3D
+## Using VKD3D-Proton
+
+The intended way to use VKD3D-Proton is as a native Win32 d3d12.dll.
+This serves as a drop-in replacement for D3D12, and can be used in Wine (Proton or vanilla flavors), or on Windows.
+
+VKD3D-Proton does not supply the necessary DXGI component. VKD3D-Proton can be used with either DXVK's DXGI implementation, or
+the Wine DXGI implementation. VKD3D-Proton implements its own IDXGISwapChain implementation when built as a native d3d12.dll.
```
Comment (Joshua-Ashton) [inline README.md:120]: s/or the Wine/or Wine's/

Comment (Joshua-Ashton) [inline README.md:120]: s/implementation//

File: README.md [hunk 5]
```diff
@@ -24,78 +61,80 @@ git clone --recursive https://github.com/HansKristian-Work/vkd3d-proton
...
+
+### A note on using VKD3D-Proton on Windows
+
+Native Windows use is mostly relevant for developer testing purposes.
+Do not expect games running on Windows 7 or 8.1 to magically make use of VKD3D-Proton
```
Comment (Joshua-Ashton) [inline README.md:125]: s/VKD3D-Proton/VKD3D-Proton,/

**Review by Joshua-Ashton — CHANGES_REQUESTED** [review]

**Comment by Joshua-Ashton** [comment]
Probably want to update the COPYING/LICENSE years also

File: README.md (hunk 2 above)
Comment (HansKristian-Work) [reply README.md:35]: Death Stranding SSE flag bug I think?

File: README.md (hunk 1 above)
Comment (HansKristian-Work) [reply README.md:30]: True, forgot about that one.

File: README.md (hunk 2 above)
Comment (HansKristian-Work) [reply README.md:35]: ye, 20.2.1 was 14th Oct, that fix was Oct 21st.

File: README.md (hunk 1 above)
Comment (HansKristian-Work) [reply README.md:30]: Fixed.

File: README.md (hunk 3 above)
Comment (HansKristian-Work) [reply README.md]: Fixed.

File: README.md (hunk 4 above)
Comment (HansKristian-Work) [reply README.md]: Fixed.

**Review by Joshua-Ashton — APPROVED** [review]
LGTM

File: README.md (hunk 2 above)
Comment (doitsujin) [reply README.md:36]: Ok.

**Review by doitsujin — APPROVED** [review]

**Pull request merged**